fvtest/compilertriltest/JitTest.hpp

```diff
@@ -135,10 +137,15 @@ template <typename T>
 std::vector<T> const_values()
    {
    return std::vector<T>{ zero_value<T>(),
+                          one_value<T>(),
+                          negative_one_value<T>(),
                           positive_value<T>(),
                           negative_value<T>(),
                           std::numeric_limits<T>::min(),
-                          std::numeric_limits<T>::max() };
+                          std::numeric_limits<T>::max(),
+                          std::numeric_limits<T>::min() + 1,
+                          std::numeric_limits<T>::max() - 1
+                        };
    }
```

Comment:
Outside the scope of this PR, but it would be useful to add int64_t arithmetic tests as well.